config/environments/test.rb

```diff
@@ -11,59 +10,52 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.cache_classes = false
-  config.action_view.cache_template_loading = true
```

Comment:
first line is similar change as other environments

second one was apparently from a spring only related comment, so we can safely delete.
https://github.com/rails/rails/commit/f75f841a2e557aedbfec63a9ad89dc262af4120a